precompiles/zk-groth16-verify/Cargo.toml

```diff
@@ -0,0 +1,69 @@
+[package]
+name = "pallet-evm-precompile-zk-groth16-verify"
+description = "A Precompile to add zk groth16 verify"
+version = "0.0.0"
+license = "Unlicense"
+publish = false
+authors.workspace = true
+edition.workspace = true
+repository.workspace = true
+
+[dependencies]
+log.workspace = true
+num_enum.workspace = true
+
+# Moonbeam
+precompile-utils.workspace = true
+
+# Substrate
+frame-support.workspace = true
+frame-system.workspace = true
+scale-codec = { package = "parity-scale-codec", workspace = true, features = ["max-encoded-len"] }
+sp-core.workspace = true
+sp-io.workspace = true
+sp-std.workspace = true
+
+# Frontier
+fp-evm.workspace = true
+pallet-evm.workspace = true
+
+# zk precompile
+hex = { version = "0.4", default-features = false }
+num-bigint = { workspace = true }
+# zksnark
+ark-bn254 = { workspace = true }
+ark-crypto-primitives = { workspace = true }
+ark-ec = { workspace = true }
+ark-ff = { workspace = true }
+ark-groth16 = { workspace = true }
```

Comment:
Would be nice to keep it in one style. I prefer .workspace = true, but it's up to you